photon-lib/src/generate/photonlib.json.in

```diff
@@ -35,19 +35,6 @@
         "linuxx86-64",
         "osxuniversal"
       ]
-    },
-    {
-      "groupId": "org.photonvision",
-      "artifactId": "photontargeting-jni",
-      "version": "${photon_version}",
-      "skipInvalidPlatforms": true,
-      "isJar": true,
-      "validPlatforms": [
-        "windowsx86-64",
-        "linuxathena",
-        "linuxx86-64",
-        "osxuniversal"
```

Comment:
We still need this, don't we? The actual .so's are in photontargeting-jni still?

Reply:
I moved the shared libraries to the -cpp artifacts (which matches the way wpilib does it). We don't want shared libraries in JARs for robot programs.

Reply:
As long as this still works in sim, I'm happy

Reply:
Works in sim on Windows 11!